### PR TITLE
feat: darkmode added to download-dropdown

### LINKED ISSUE #329: [Fe] dark mode enhancement in download page
**Actual Behaviour**
In dark mode
Dialogue appearing after clicking Download button in download page does not follow dark mode

**Expected Behaviour**
Dialogue box should also be following dark mode

**Steps to reproduce it**

- Goto apisix.apache.org -> click downloads -> click download button 

**Screenshots of the issue**

scrnli_04_05_2021_14-42-52

**Would you like to work on the issue?**

yes i would like to work in this issue,  but as this could be the good first issue , i would like new committers to give a PR :smiley:

### PR DESCRIPTION
**Pull request by iamayushdas**
Fixes: #329 

Changes:

Added darkmode for download button -> dropdown menu

Screenshots of the change:

Earlier:
Screenshot from 2021-05-10 12-10-53

Now:
Screenshot from 2021-05-10 12-09-28

### CODE REVIEW

**Review by imjoey — APPROVED** [review]
The deployed version looks good to me.
@iamayushdas Thanks.

**Comment by iamayushdas** [comment]
> The deployed version looks good to me.
> @iamayushdas Thanks.

your welcome :smile:

**Comment by Yiyiyimu** [comment]
image

Currently the color change at hover is too small for user to make sure where is s/he

**Comment by iamayushdas** [comment]
> image
> 
> Currently the color change at hover is too small for user to make sure where is s/he

OKAY ,thanks for reviewing

**Comment by iamayushdas** [comment]
> image
> 
> Currently the color change at hover is too small for user to make sure where is s/he

is it fine now?

File: website/src/css/customTheme.css [hunk 1]
```diff
@@ -644,4 +656,4 @@ a:hover {
 
 .help-page .card p {
   color: var(--color-help-subtitle);
-}
+}
```
Comment (Yiyiyimu) [inline website/src/css/customTheme.css]: missing a newline here. Maybe you could make your IDE do the check

Comment (juzhiyuan) [inline website/src/css/customTheme.css]:
```suggestion
}

```

**Comment by iamayushdas** [comment]
@Yiyiyimu new line has been added, have a look now

**Review by Yiyiyimu — APPROVED** [review]

**Pull request merged**